saleor/graphql/core/filters.py

```diff
@@ -0,0 +1,12 @@
+import django_filters
+
+
+class EnumFilter(django_filters.CharFilter):
+    """
+    Filter for GraphQL's enum objects. enum_class stores graphQL enum
```

Comment:
There are a few grammar issues in this docstring, maybe @timcla help with this (or Grammarly if he's busy)?